package.json

```diff
@@ -52,10 +52,12 @@
   "dependencies": {
     "@ensdomains/address-encoder": "1.1.1",
     "@ensdomains/content-hash": "^3.0.0-beta.5",
-    "@ensdomains/ens-contracts": "1.4.0",
-    "@ensdomains/ens-test-env": "0.6.0",
+    "@ensdomains/ens-contracts": "1.4.0-beta.2",
+    "@ensdomains/ens-test-env": "github:namehash/ens-test-env",
```

Comment:
Could you please link to the branch/commit instead of linking to github? This way I can patch it back to test-env. If this is a PR for integrating with ensnode, just do a `pnpm link ../ens-test-env` instead